templates/al2/template.json

```diff
@@ -208,6 +208,18 @@
         "WORKING_DIR={{user `working_dir`}}"
       ]
     },
+    {
+      "type": "shell",
+      "remote_folder": "{{ user `remote_folder`}}",
+      "script": "{{template_dir}}/../shared/provisioners/cache-pause-container.sh",
+      "environment_vars": [
+        "AWS_ACCESS_KEY_ID={{user `aws_access_key_id`}}",
+        "AWS_SECRET_ACCESS_KEY={{user `aws_secret_access_key`}}",
+        "AWS_SESSION_TOKEN={{user `aws_session_token`}}",
+        "AWS_REGION={{user `aws_region`}}",
+        "PAUSE_CONTAINER_CMD=echo $(/etc/eks/get-ecr-uri.sh \"{{user `aws_region`}}\" \"$(imds 'latest/meta-data/services/domain')\")/eks/pause:3.5"
```

Comment:
i know what this looks like but hear me out 🤣 , the plumbing to get this script (which is also the map of region to accounts) is messy to sync between al2 and al23. i opted to just expose the mapping from `nodeadm` and allow each distro to pass in their expected handler code.

Reply:
I do think this is kinda wonky, what if we moved the shared logic into a utility that could be used by a simple provisioner in each template?

Reply:
good with this, ill run another test sometime soon